_layouts/showcase-item.html

```diff
@@ -141,12 +141,18 @@ <h3 style="opacity: 0.75; margin-top: 0.5rem">
 				</a>
 				{% endif %}
 
-				{% if page.xbox %}
-				<a href="{{ page.xbox }}" class="btn" style="margin: 0 0.5rem; white-space: nowrap" target="_blank">
+				{% if page.microsoft %}
```

Comment:
Any reason for the rename? Microsoft themselves are using xbox as the brand for the gaming side of things, and the url used in Primal Light is also an xbox URL `https://www.xbox.com/en-US/games/store/primal-light/9N2XZFRWG6R8`

Reply:
Yes, the reason is the name collision:

> I also added templates for various collection items, so we can have a good view on their format, parameters, and just create new docs with more ease. This actually exposed that showcase items used page.xbox for two separate purposes, as a bool and as a store link. I fix this "shadowing" issue, but we may want to scope those flags in the future. Also added support for PlayStation store links.

I think a proper solution would be to use objects and group those properties, but for now a simple renaming fix should be good enough.